sci-libs/nipype/nipype-9999.ebuild

```diff
@@ -18,12 +18,14 @@ EGIT_REPO_URI="https://github.com/nipy/nipype"
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS=""
-IUSE=""
+IUSE="fsl spm"
 
 DEPEND="
 	dev-python/setuptools[${PYTHON_USEDEP}]
 	dev-python/numpy[${PYTHON_USEDEP}]
-	>=sci-libs/nibabel-1.2[${PYTHON_USEDEP}]"
+	>=sci-libs/nibabel-1.2[${PYTHON_USEDEP}]
+	fsl? ( sci-biology/fsl )
+	spm ( sc-biology/spm )"
```

Comment:
These things could be prevented by running `repoman` (inside `sci-libs/nipype`).